last-donation/last-donation-settings.js

```diff
@@ -3,8 +3,10 @@ ELT.settings = {
 	animate: true,
 	// animation pause duration
 	animationPauseMS: 5000,
+	// direction of animation
```

Comment:
So `left` means comes in from the left, or goes to the left?

Reply:
Goes to the left. I thought that might cause some confusion. Thoughts on a better name?

Reply:
`animateFrom`?

Reply:
Changed to `animateTo`